dotnetclr/setup.py

```diff
@@ -16,7 +16,7 @@
     long_description = f.read()
 
 
-CHECKS_BASE_REQ = 'datadog_checks_base>=4.2.0'
+CHECKS_BASE_REQ = 'datadog_checks_base>=11.4.0'
```

Comment:
Should we update the other PRs as well to include that?

Reply:
No, this one is because it's a PDH check and needs changes recently included to the base class to be able to not pass agentConfig